packages/MarkdownEditor-Core.package/MarkdownEmphasisDelimiter.class/instance/attributeFrom.to.delimiterLength..st

```diff
@@ -0,0 +1,6 @@
+attributes
+attributeFrom: aStartPosition to: anEndPosition delimiterLength: aLength
```

Comment:
```suggestion
attributeFrom: startPosition to: endPosition delimiterLength: aLength
```
We do variable shadowing elsewhere. If this bothers you, I'd prefer `attributeStartPosition`

Reply:
I don't like `attributeStartPosition`. We know that everything we do in this method is regarding attributes. Thus, this would be redundant. Yeah, we tried to fix the shadowing elsewhere, too.